src/foam/doc/AxiomListView.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'foam.doc',
+  name: 'AxiomListView',
+  extends: 'foam.u2.View',
+  requires: [
+    'foam.doc.Axiom',
+    'foam.doc.AxiomLink',
+    'foam.doc.dao.AxiomDAO',
+    'foam.mlang.sink.Count',
+  ],
+  implements: [
+    'foam.mlang.Expressions',
+  ],
+  properties: [
+    {
+      class: 'String',
+      name: 'modelID',
```

Comment:
I think naming convention is modelId not modelID

Reply:
done